ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -139,9 +139,7 @@ def wrapped(*args, **kwargs):
 
 def prepare_reverse_zone(host, ip):
     zone = get_reverse_zone_default(ip)
-    result = host.run_command(["ipa",
-                      "dnszone-add",
-                      zone], raiseonerr=False)
+    result = add_dns_zone(host, zone, skip_overlap_check=True)
```

Comment:
``add_dns_zone()`` returns None and fails when the zone doesn't exist and can be added.